### PR TITLE
[Stevenage] Update GovMetric and year.

### LINKED ISSUE #84: Govmetric changes for Stevenage
Email from client -->
 
We are changing our geometric code for our websites and I am told you host the petitions site for us?
 
http://petitions.stevenage.gov.uk/list
 
Will you be able to add the supplied code to the website for us please and change the copyright at the bottom to be 2017 rather than 2011?
[GovMetric.txt](https://github.com/mysociety/petitions/files/1193900/GovMetric.txt)

### PR DESCRIPTION
**Pull request by dracos**
Fixes #84

### CODE REVIEW

File: templates/stevenage/foot.html [hunk 1]
```diff
@@ -15,33 +15,46 @@
...
+    <a
+        href="https://websurveys.govmetric.com/theme/gm/1340?Q_RATINGID=4"
+        target="_blank"
+        class="gm_sidebar_anchor"
+        title="rate this page as POOR"
+        rel="nofollow">
+        <img style="width:30px;height:30px;" src="https://wsstatic.govmetric.com/imgs/smileys/large/poor.png" alt="red smiley (poor)" />
+    </a>
+
+                <p class="inst"><img src="/Images/rateThisPage.gif" alt="Please rate this page." />
```
Comment (davea) [inline templates/stevenage/foot.html]: This image doesn't exist locally, it probably wants the `http://www.stevenage.gov.uk/` prefix.

File: templates/stevenage/foot.html [hunk 2]
```diff
@@ -15,33 +15,46 @@
...
-					<input type="hidden" name="organisationid" value="343" />
-					<input type="image" src="http://www.govmetric.com/images/green-face-30x30.gif" alt="Rate as Good" name="metric1" value="1" />
-					<input type="image" src="http://www.govmetric.com/images/amber-face-30x30.gif" alt="Rate as Average" name="metric2" value="2" />
-					<input type="image" src="http://www.govmetric.com/images/red-face-30x30.gif" alt="Rate as Poor" name="metric3" value="3" />
-					<p class="inst"><img src="http://www.stevenage.gov.uk/Images/rateThisPage.gif" alt="Please rate this page." /></p>
-				</div>
+<!-- WebMetric snippet starts here -->
+
+<div class="gm_sidebar">
+    <a
```
Comment (davea) [inline templates/stevenage/foot.html]: For some reason these images don't appear on Safari for me (http://davea.petitions.dev.mysociety.org):
screen shot 2017-08-02 at 16 03 53

However, they're fine on Chrome. Not sure what's going on there.
screen shot 2017-08-02 at 16 03 47

Comment (dracos) [inline templates/stevenage/foot.html]: Ad blocker running on Safari?

Comment (davea) [inline templates/stevenage/foot.html]: Hah, yes. As you were 😄

**Pull request merged**